test.sh

```diff
@@ -21,48 +25,135 @@ function prepareOutputDir() {
 }
 
 function createTestEnvironment() {
+  echo '-----------------------test.sh------------------------------'
   echo "Creating test environment"
-  echo '-----'
+  echo '------------------------------------------------------------'
 
   # Build test container & start the cluster
-  docker-compose pull postgres possum
-  docker-compose build --pull test
-  docker-compose up -d client possum postgres
+  docker-compose pull client conjur postgres
+  docker-compose build --pull test test-https conjur-proxy-nginx
```

Comment:
You can condense this and the line above it into

```
docker-compose build --pull
```
compose will then build all containers that need built and pull new images for ones that don't.

Reply:
I will change it :)